src/mlang/backend_compilers/bir_to_dgfip_c.ml

```diff
@@ -524,25 +540,34 @@ let generate_c_program (program : Bir.program)
   let header_filename = Filename.remove_extension filename ^ ".h" in
```

Comment:
Shall we add the newly exposed functions to the header?

Reply:
done